### DISCUSSION TITLE
Changes in the translation team

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enrouteText#34] [Team Posts]
Dear Translators,

after one year of supporting our project, Luca Bertoncello has decided to leave the translation team for private reasons.

@lucabert That was an incredible job. Thank you for all your help! I have moved your name to the "Alumni" section of the "Author dialog". I would remove your account from the translation team in a few days.

@Ivan081 You are now the only translator for the Italian language. If you like, I could add a sentence to the "What's new" dialog, asking for help with Italian translations. Simply send me an e-mail or reply to this message.

Best,

Stefan.

**Comment by Ivan081** [comment]
Yes I have already seen in the translations that you asked for help for
Italian ... these days I am doing flight tests for that I am late with the
translations

Il giorno mar 4 mag 2021 alle ore 08:12 Stefan Kebekus <
***@***.***> ha scritto:
